modules/managers/languages/en.json

```diff
@@ -0,0 +1,31 @@
+{
+    "MANAGERS": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "Manager",
+            "PLURAL_NAME": "Managers"
+        },
+        "FIELDS": {
```

Comment:
Add Japanese (ja.json), Korean (ko.json), and Simplified Chinese (zh_cn.json) JSONs for `Mangers` module